app/design-system/design-system-internals/src/main/kotlin/com/hedvig/android/design/system/internals/SnackBar.kt

```diff
@@ -0,0 +1,158 @@
+package com.hedvig.android.design.system.internals
+
+import androidx.compose.foundation.clickable
+import androidx.compose.foundation.layout.Arrangement
+import androidx.compose.foundation.layout.Row
+import androidx.compose.foundation.layout.Spacer
+import androidx.compose.foundation.layout.fillMaxWidth
+import androidx.compose.foundation.layout.padding
+import androidx.compose.foundation.layout.width
+import androidx.compose.material3.Snackbar
+import androidx.compose.material3.SnackbarHost
+import androidx.compose.material3.SnackbarHostState
+import androidx.compose.material3.SnackbarResult.ActionPerformed
+import androidx.compose.material3.SnackbarResult.Dismissed
+import androidx.compose.material3.Text
+import androidx.compose.runtime.Composable
+import androidx.compose.runtime.LaunchedEffect
+import androidx.compose.runtime.remember
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.draw.clip
+import androidx.compose.ui.graphics.Color
+import androidx.compose.ui.graphics.Shape
+import androidx.compose.ui.text.TextStyle
+import androidx.compose.ui.text.style.TextDecoration
+import androidx.compose.ui.unit.dp
+
+@Composable
+fun InternalSnackBar(
+  snackbarText: String,
+  showSnackbar: Boolean,
+  showedSnackbar: () -> Unit,
+  colors: NotificationColors,
+  shape: Shape,
+  textStyle: TextStyle,
+  icon: @Composable () -> Unit,
+  modifier: Modifier = Modifier,
+  action: (() -> Unit)? = null,
+  actionLabel: String? = null,
+) {
+  val snackbarHostState = remember { SnackbarHostState() }
+  LaunchedEffect(showSnackbar, snackbarText) {
+    if (!showSnackbar) return@LaunchedEffect
+    val result = snackbarHostState.showSnackbar(snackbarText, actionLabel = actionLabel)
+    when (result) {
+      Dismissed -> {
+        showedSnackbar()
+      }
+
+      ActionPerformed -> {
+        if (action != null) {
+          action()
+        }
+        showedSnackbar()
+      }
+    }
+  }
+  SnackbarHost(snackbarHostState, modifier.padding(16.dp)) { _ ->
```

Comment:
The 16.dp padding here looks to be added on top of some other padding too. On my phone, when I turned this to 0.dp it looked even more correct I'd say. perhaps the component has a padding itself applied for the insets + some sane default.
I'd suggest you try removing this padding modifier alltogether and see how that looks to you too

Reply:
On my phone it goes then to the bounds of the screen. I'd say let's leave it to be sure:)